src/components/clusters/utils.ts

```diff
@@ -6,7 +6,8 @@ import {
   OperatorCreateParams,
 } from '../../api/types';
 
-export const isSingleNodeCluster = (cluster: Cluster) => cluster.highAvailabilityMode === 'None';
+export const isSingleNodeCluster = (cluster: Partial<Cluster>) =>
```

Comment:
not needed and confusing

Reply:
I agree, I don't thinks this helper is going to be ever used by passing another object than a cluster as a parameter.